include/swift/AST/ASTContext.h

```diff
@@ -1107,6 +1115,9 @@ class ASTContext final {
   /// If a module by this name has already been loaded, the existing module will
   /// be returned.
   ///
+  /// \param ModulePath The module's \c ImportPath which describes
+  /// the name of the module being loaded, possibly including submodules.
+
```

Comment:
Outdated?

Reply:
Nah, the ModulePath argument was already here originally, so I added some documentation for it.